format.sh

```diff
@@ -20,7 +20,7 @@ fi
 
 echo "Formatting C files. This will take a bit"
 echo "Running clang-format..."
-clang-format-11 ${FORMAT_OPTS} src/**/*.c
+clang-format ${FORMAT_OPTS} src/**/*.c
```

Comment:
Very nice work. My only question is does removing the -11 mean when this is run, it uses the default clang-format on everyone's computer? Would this lead to clang version clashes when formats on different computers apply? (I actually have no idea, that's just a guess/question).

Reply:
Yeah, I don't think this is a good solution since we've seen different versions produce different result. We should standardise on a version, the question being, which?

Reply:
I agree with Elliptic. I'd prefer to standardize on a specific version in order to avoid different formatting conflicts. Perhaps this should be discussed over in Discord.

Reply:
My personal preference would be to standardize a version, and use the installation instructions as a place to help people install the correct version.

Reply:
We'll use `clang-format-11` as discussed in Discord.